### PR TITLE
Fix IE11 styling

### PR DESCRIPTION
**Pull request by hueitan**
Phabricator Ticket : https://phabricator.wikimedia.org/T260781

Problem
===

IE11 has a lot of styling issues

Solution
===

 - [x] css value none doesn't existed in IE11
 - [x] Image Slider position

### CODE REVIEW

**Comment by hueitan** [comment]
this update is mainly for IE11, but still affect any other browser, it's worth to check again in chrome/firefox...

File: style/gallery.less [hunk 1]
```diff
@@ -60,27 +58,24 @@
...
 		display: flex;
-		display: -ms-flex;
 		max-width: 100%;
 		height: 100%;
 	}
 
 	&-slider {
-		display: inline-flex;
-		display: -ms-inline-flex;
+		display: flex;
```
Comment (medied) [inline style/gallery.less:66]: Hopefully these won't conflict too much with #85 or #86. Working well in Firefox and Chrome for me right now!

**Comment by medied** [comment]
So basically we should stay away from `unset` moving forward

**Review by medied — APPROVED** [review]

File: style/gallery.less (hunk 1 above)
Comment (hueitan) [reply style/gallery.less:66]: good news 🎉 I have tested merging the others (only 1 conflict which easy to fix), works fine 💯

**Comment by stephanebisson** [comment]
> So basically we should stay away from `unset` moving forward

See #87

**Review by stephanebisson — APPROVED** [review]

**Pull request merged**